WooCommerce/Classes/ViewRelated/Dashboard/Dashboard.storyboard

```diff
@@ -220,9 +220,12 @@
                         <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
-                            <textView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" textAlignment="natural" translatesAutoresizingMaskIntoConstraints="NO" id="VZ4-xD-33I">
+                            <textView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" editable="NO" textAlignment="natural" selectable="NO" translatesAutoresizingMaskIntoConstraints="NO" id="VZ4-xD-33I">
```

Comment:
do we still want to make the text selectable so that they could select a part of the logs and share? or they should just share the whole logs with the share navigation button?

Reply:
Hmmm, yes. Good point. The WPiOS app allows the Copy ability for selected text. Will fix!